### PR TITLE
Moved create databse recipe to test cookbook

### PR DESCRIPTION
**Pull request by fahlmant**
Cleaning up outdated code a bit. This recipe is still needed for test kitchen, so I moved it to a test cookbook.

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -2,4 +2,5 @@ source 'https://supermarket.chef.io'
 
 cookbook 'haproxy'
 cookbook 'nodejs-webapp', git: 'git@github.com:osuosl-cookbooks/nodejs-webapp'
+cookbook 'timesync-test', path: './test/integration/cookbooks/timesync-test'
```
Comment (ramereth) [inline Berksfile]: Please remove the `./`

**Comment by ramereth** [comment]
:+1:

**Comment by fahlmant** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v0.2.1.  Have a nice day!

**Pull request merged**